core/src/main/java/cucumber/runtime/model/CucumberFeature.java

```diff
@@ -1,83 +1,99 @@
 package cucumber.runtime.model;
```

Comment:
This package has become quite anemic. All closely tied to RuntimeOptions. Might want to consolidate this, RuntimeOptions and friends into a options package.